backend/src/models/user.js

```diff
@@ -6,12 +6,12 @@ class User {
     }
 
     /**
-     * TODO Return specialsed information for auth'd user
+     * Return specialised information for auth'd user
      * @param {string} id The id of the auth'd user
      */
     getProfile(id) {
         return this.db
-            .query('SELECT * FROM user WHERE id=?', [id])
+            .query('SELECT id, email, displayName, degree, gradYear, description, picture, joined FROM user WHERE id=?', [id])
```

Comment:
curious what picture is being stored as? Base64?

Reply:
it's just a URL for now :)

Reply:
kk may not ever be used XD